src/pages/en/guides/cms/netlify-cms.mdx

```diff
@@ -15,6 +15,8 @@ service: Netlify CMS
 
 - Blog Post: [Author your Astro site's content with Git-based CMSs](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) by Aftab Alam 
 
+- Youtube Tutorial: [Create a Custom Blog with Astro & NetlifyCMS in MINUTES!](https://www.youtube.com/watch?v=3yip2wSRX_4&t=319s) by Kumail Pirzada
```

Comment:
Thanks for this @JS-GitRepo! I think we don’t want the `t` parameter to jump to the middle of the video:

```suggestion
- Youtube Tutorial: [Create a Custom Blog with Astro & NetlifyCMS in MINUTES!](https://www.youtube.com/watch?v=3yip2wSRX_4) by Kumail Pirzada
```